packages/augur-ui/src/modules/reporting/containers/reporting-bonds-view.ts

```diff
@@ -14,12 +14,12 @@ const mapStateToProps = (state: AppState, ownProps) => {
   const userFunds = state.appStatus.gsnEnabled ? createBigNumber(loginAccount.balances && loginAccount.balances.dai || ZERO) : createBigNumber(loginAccount.balances && loginAccount.balances.eth || ZERO);
   const hasForked = !!state.universe.forkingInfo;
   const migrateRep =
-    hasForked && universe.forkingInfo.forkingMarket === market.id;
+    hasForked && universe.forkingInfo && universe.forkingInfo.forkingMarket === market.id;
```

Comment:
I believe this:
`universe.forkingInfo && universe.forkingInfo.forkingMarket`
Might be replaced by:
`universe.forkingInfo?.forkingMarket`